java/client/src/org/openqa/selenium/remote/tracing/AttributeKey.java

```diff
@@ -0,0 +1,56 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.remote.tracing;
+
+public enum AttributeKey {
```

Comment:
It's likely that we can't know all the possible keys in advance. An `enum` is a poor choice given this constraint, so it may be better to use an `interface`

Reply:
+1 , it is not possible to know keys in advance. AttributeKey enum is to store keys which are used repeatedly and which are part of OpenTelemetry specification, all attribute key do not need to be part of the enum. Do you still suggest to use an interface instead?